frontend/tests/acceptance/pages/notesPage.js

```diff
@@ -0,0 +1,61 @@
+const { I } = inject();
+
+module.exports = {
+  fields: {
+    title: '//label[contains(text(),"Title")]/parent::div//input[contains(@class,"MuiInputBase-input")]',
+    content: '//label[contains(text(),"Content")]/parent::div//textarea',
+  },
+  elements: {
+    addnote_button: '//button/span[contains(text(),"Add")]',
+    updatenote_button: '//button/span[contains(text(),"Update")]',
+    deletenote_button: '//div[contains(@class,"makeStyles-cardAction")]//*[name()="svg"]',
+    openaddnote_dialog: '//div[contains(@class,"makeStyles-addBox")]',
+    openeditnote_dialog: '//div[contains(@class,"makeStyles-cardContent")]',
+    notesContainer: '//div[contains(@class,"makeStyles-notes")]',
+    getNoteContext(title) {
+      return `//h6[contains(text(),"${title}")]/ancestor::div[contains(@class,"makeStyles-card")]`;
+    },
+  },
+  addNote(note) {
+    I.waitForElement(this.elements.openaddnote_dialog, 5);
```

Comment:
use a common variable with reasonable comment while using such hard_coded values

Reply:
didn't understand the context.